src/hardware-security-modules/HISTORY.rst

```diff
@@ -4,7 +4,7 @@ Release History
 ===============
 0.2.0
 ++++++
-* 'az dedicated-hsm': add parameters '--mgmt-network-subnet' and '--mgmt-network-interfaces' and
+* 'az dedicated-hsm': add parameters '--mgmt-network-subnet' and '--mgmt-network-interfaces'
 * Bump up SDK to 2021_11_30
```

Comment:
Remove the note of this line as API version used in SDK is an implementation detail that should not be exposed to customers.